rules_nodejs.patch

```diff
@@ -0,0 +1,13 @@
+diff internal/pkg_npm/packager.js internal/pkg_npm/packager.js
```

Comment:
What does the patch do?

Reply:
I've never used pkg_npm in a repo that uses `v` prefix on the tags. NPM package versioning has to be purely numeric so it needs to be stripped. I'll make the same change upstream.